src/common/styles/flex.css

```diff
@@ -0,0 +1,3 @@
+.displayFlex{
```

Comment:
I want to add some common flex CSS to add to global to make it easier to arrange all of our elements the same way. 
Will cover more in fonts.css